deeposlandia/inference.py

```diff
@@ -150,7 +150,8 @@ def init_model(problem, instance_name, image_size, nb_labels, dropout, network):
 
 def predict(filenames, dataset, problem, datapath="./data", aggregate=False,
             name=None, network=None, batch_size=None, dropout=None,
-            learning_rate=None, learning_rate_decay=None, output_dir=None):
+            learning_rate=None, learning_rate_decay=None,
+            output_dir="/tmp/deeposlandia/predicted"):
```

Comment:
Is the folder `/tmp/deeposlandia` created if it does not exist?

Reply:
It is created in `webapp.py`, as this module calls `inference.predict(...)` several times. However, I can add a `os.makedirs` statement to guarantee the folder creation, for instance when the method is called in another context.